plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/ui/DBPPlatformUI.java

```diff
@@ -136,6 +136,12 @@ DBPAuthInfo promptUserCredentials(
      */
     String promptProperty(String prompt, String defValue);
 
+    /**
+     * Ask user to enter a multiline value
+     */
+    @NotNull
+    String promptText(String title, String prompt, String defValue);
```

Comment:
```suggestion
    String promptText(@Nullable String title, @Nullable String prompt, @Nullable String defValue);
```

Reply:
This method _should_ probably be marked `@Nullable` as it may return `null` if no text was provided.